src/App.cpp

```diff
@@ -13,22 +13,14 @@
 
 #include "gvizdoom/DoomGame.hpp"
 
-#include <opencv2/highgui.hpp>
-
-
 using namespace doot2;
 using namespace gvizdoom;
 
-
```

Comment:
No need to remove the 2nd space, it's used to separate semantically different sections in files (includes, definitions, namespaces, functions)